src/ontology/cl-edit.owl

```diff
@@ -21853,21 +21852,25 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0008036 "extravillous cytotr
 AnnotationAssertion(rdfs:label obo:CL_0008036 "extravillous trophoblast"@en)
 SubClassOf(obo:CL_0008036 obo:CL_0000351)
 
-# Class: obo:CL_0008037 (obo:CL_0008037)
+# Class: obo:CL_0008037 (gamma motor neuron)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersion:Gamma_motor_neuron&oldid=906640465") obo:IAO_0000115 obo:CL_0008037 "A lower motor neuron that innervates only intrafusal muscle fibers.")
-AnnotationAssertion(<http://www.w3.org/2004/02/skos/core#prefLabel> obo:CL_0008037 "gamma motor neuron"@en)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1093/acprof:oso/9780195159561.001.1") Annotation(oboInOwl:hasDbXref "WikipediaVersion:Gamma_motor_neuron&oldid=906640465") obo:IAO_0000115 obo:CL_0008037 "A lower motor neuron that innervates only intrafusal muscle fibers. Unlike the alpha motor neurons, gamma motor neurons do not directly adjust the lengthening or shortening of muscles but function in adjusting the sensitivity of muscle spindles and in keeping muscle spindles taut, thereby allowing the continued firing of alpha neurons.")
```

Comment:
Is this how you add a DOI? the manual came with PMID, this only has a DOI or ISBN-13